lab5.py

```diff
@@ -0,0 +1,83 @@
+'''
+Імпортує клас енам
+'''
+
+
+from enum import Enum
+
+class PlanetType(Enum):
+    '''
+    Клас по визначенню типу планети
+    '''
+    TERRESTRIAL = 1
+    JOVIAN = 2
+
+
+class Planet:
+    '''
+    Клас який містить дані про планету
+    '''
+    def __init__(self, name, kg, speed, degree, hour, kilometres, planet_type):
+        '''
+        Функція яка ініціалізує початкові значення
+        '''
+        self.name = name
+        self.mass = kg
+        self.orbital_velosity = speed
+        self.mean_temparature = degree
+        self.length_of_day = hour
+        self.distance_from_sun = kilometres
+        self.type = planet_type
+
+    def find_mass(self):
+        '''
+        Виводить масу планети
+        '''
+        return f"Маса планети {self.name}: {self.mass}"
+
+    def find_mean_temparature(self):
+        '''
+        Виводить температуру планети
+        '''
+        return f"Температура планети {self.name}: {self.mean_temparature}"
+
+
+# Забрати наслідування, все крім селф в ініті і забрати супер ініт
+class Planetary(Planet):
+    '''
+    Клас який містить планети
+    '''
+    def __init__(self, name, kg, speed, degree, hour, kilometres, planet_type):
+        '''
+        Функція яка ініціалізує планети
+        '''
+        super().__init__(name, kg, speed, degree, hour, kilometres, planet_type)
+        self.planets = []
+
+    def add_planet(self, planet):
+        '''
+        Функція яка додає планети
+        '''
+        self.planets.append(planet)
+
+    def sort_by_length_of_day(self):
+        '''
+        Функція яка сортує планети за довжиною дня
+        '''
+        self.planets.sort(key=lambda x: x.length_of_day)
+        for i in self.planets:
+            print(f"Назва планети: {i.name}, Довжина світлового дня: {i.length_of_day}")
```

Comment:
It would be better if you replace the print with a return and implement print in main file

Reply:
i can't do this

Reply:
done